spec.md

```diff
@@ -642,13 +642,25 @@ as `[<registry>/][<project>/]<image>[:<tag>|@<digest>]`.
     image: my_private.registry:5000/redis
 ```    
 
-If the image does not exist on the platform, Compose implementations MUST attempt to pull it. Compose
+If the image does not exist on the platform, Compose implementations MUST attempt to pull it based on the `pull_policy`. Compose
 implementations with build support MAY offer alternative options for the end user to control precedence of
-pull over building the image from source, however pulling the image MUST be the default behaviour.
+pull over building the image from source, however pulling the image MUST be the default behavior.
 
 `image` MAY be omitted from a Compose file as long as a `build` section is declared. Compose implementations
 without build support MUST fail when `image` is missing from the Compose file.
 
+
+### pull_policy
+
+`pull_policy` defines the decisions Compose implementations will make when it starts to pull images. Possible values are:
+
+* `always`: Compose implementations SHOULD always pull the image from the registry ignoring other caches.
```

Comment:
Is there a reason for ignoring all caches? This is not necessary given the design of the OCI images. Even if the image has changed it may only have changed the last layer so the previous layers can be re-used from a local cache. According to the image spec that should be the default IIRC. Would be nice to clarify this. Not all implementations have to copy what is used in Kubernetes 1 to 1. Especially if a behavior can cause massive DDOS storms if used incorrectly.

Reply:
Personally i don't think this means ignore all caches

Reply:
Coming from the `docker pull` perspective, I read "pull the image from the registry" to mean fetch the manifest for the tag from the registry and if it doesn't match the node cache then pull the layers that diff.

We may need to adjust the wording to make the intention more clear.